labkey/utils.py

```diff
@@ -31,3 +31,44 @@ def default(self, o):
 def json_dumps(*args, **kwargs):
     kwargs.setdefault("cls", DateTimeEncoder)
     return json.dumps(*args, **kwargs)
+
+
+def transform_helper(userTransformFunc, filePathRunProperties):
+    #filePathRunProperties must be explicitly defined as ${runInfo} within the user's transform script
+    #parse run properties to for results data in and out filepaths
+    filePathIn = ''
```

Comment:
In Python the generally accepted naming convention is to use `snake_case` for variable names, not `camelCase`, and `PascalCase` for classes. Can you convert these variables to be `snake_case`?